### PR TITLE
Super user screens (now with a fix)

### PR DESCRIPTION
**Pull request by dariusk**
New PR for the fix to the errors on live. The problem was our `helpers.js` function `getSuperAdminMask()`.

This is a tricky one -- `localStorage` only stores strings, and casts `true` to `'true'` and `false` to `'false'`. So you might think that checking `superAdminMaskSession !== 'false'` would tell you if it was true, but in fact you need `superAdminMaskSession === 'true'` -- this is because in the case where `superAdminMaskSession` is not set at all, you get `null`. 

So regular users who had never touched `superAdminMaskSession` in their local storage would get `null`, which does not equal `'false'`, so the code thought the screen should show because `'false' !== null`. This is why regular users were seeing the super admin screen on content (but not the super admin controls at the bottom!)

### CODE REVIEW

**Review by amoedoamorim — CHANGES_REQUESTED** [review]
Great catch @dariusk ! It looks like this bug was introduced by the addressing of my review of the original PR ! :facepalm:  Please just fix this minor casing issue.

File: src/app/helpers.js [hunk 1]
```diff
@@ -384,6 +384,18 @@ function getTimeZoneOptions() {
...
+/**
+ * Return a boolean for super admin screen page
+ */
+function getSuperAdminMask(state) {
+  const { superAdminMask, superAdminMaskAction } = state;
+  if (superAdminMaskAction === 'page') {
+    return superAdminMask;
+  }
+  const superAdminMaskSession = sessionStorage.getItem('superAdminMaskSession');
+  return superAdminMaskSession === 'true';
```
[the remark this answers is not in the document]
Comment (melsawy) [reply src/app/helpers.js]: no I should keep !== 'false' but handle a case for non super admin users

File: src/app/components/media/SuperAdminControls.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+import Button from '@material-ui/core/Button';
+import { FormattedMessage } from 'react-intl';
+import styles from './SuperAdminControls.module.css';
+
+const SuperAdminControls = ({
+  handleSuperAdminMask,
+  handleSuperAdminMaskSession,
+}) => {
+  const superAdminMaskSession = sessionStorage.getItem('superAdminMaskSession');
+  const maskSession = superAdminMaskSession !== 'false';
```
[the remark this answers is not in the document]
Comment (melsawy) [reply src/app/components/media/SuperAdminControls.js:13]: I kept it and handle a non super admin users https://github.com/meedan/check-web/pull/1527/commits/03eed84f7fe4a4eea49555974af29a46302dec65

File: src/app/helpers.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (dariusk) [reply src/app/helpers.js]: @amoedoamorim This does work given the commit that @melsawy linked. The bug had to do with the initial state of `localStorage` for non-superadmins, but Sawy's code skips `localStorage` entirely and assumes `false` if you're not a superadmin. I tested locally in a clean browser with a non-superadmin user and it does seem to work as designed.

File: src/app/components/media/SuperAdminControls.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (dariusk) [reply src/app/components/media/SuperAdminControls.js:13]: Tested locally and it works well given the routing around `localStorage` for non-superadmins.

**Comment by dariusk** [comment]
@amoedoamorim I can't approve my own PR but let me know if my explanation didn't make sense!

**Review by amoedoamorim — APPROVED** [review]

**Pull request merged**